yesod-bin/yesod-bin.cabal

```diff
@@ -1,5 +1,5 @@
 name:            yesod-bin
-version:         1.6.0.6
+version:         1.6.1
```

Comment:
Can you also include a ChangeLog entry?

Reply:
Of course! I planned to do it after the other changes had been finalised but it's done now.